### DISCUSSION TITLE
Add php-soap extension to nextcloud

### DISCUSSION THREAD

**Discussion by arvanus** [nextcloud/all-in-one#475] [Ideas] [answered]
I'm trying to run `nextclound `with `aio`, and found this app: https://github.com/SpicyWeb-de/nextcloud-user-ispconfig that requires `soap`, is it possible to add php-soap support, or/else how can I do it? 
*When I try to install via `occ` I get this error:
```
bash-5.1$ /var/www/html/occ app:enable nextcloud-user-ispconfig
App "User support for ISPConfig mail accounts" cannot be installed because the following dependencies are not fulfilled: The library soap is not available.
```
Thanks!

**Comment by szaimen** [comment]
Hello, 

unfortunately we cannot provide all needed dependencies for all apps out there because it would make the project unmaintainable.
So I fear this app is out of scope for this project.

However you might be able to add the required dependency to the Nextcloud container by building the containier manually using https://github.com/nextcloud/all-in-one/tree/main/manual-install.

**Comment by szaimen** [comment, marked as answer]
This will be handled with https://github.com/nextcloud/all-in-one/issues/1162

**Reply by szaimen** [reply]
This is now released with v3.0.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel
